libs/filamat/src/eiff/Flattener.h

```diff
@@ -125,6 +125,18 @@ class Flattener {
         mCursor += 4;
     }
 
+    // This first writes "n" (which is a single byte), followed by "n" zeroes.
+    // The subsequent write is guaranteed to be on a 8-byte boundary.
+    void writeAlignmentPadding() {
```

Comment:
It seems even if we're already on an 8-byte boundary, we still have to write 8 bytes of padding (a 7 plus 7 zeroes). Maybe the reader could be slightly more intelligent and automatically align reads to 8 bytes to avoid the wasted space.

Reply:
That's brilliant!  Added a commit.